test/pstl_offload/memory/usm_memory_replacement_system_alloc.cpp

```diff
@@ -0,0 +1,25 @@
+// -*- C++ -*-
+//===----------------------------------------------------------------------===//
+//
+// Copyright (C) 2023 Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#if !__SYCL_PSTL_OFFLOAD__
+#error "PSTL offload compiler mode should be enabled to run this test"
+#endif
+
+// in this translation unit we use system allocations while compiling with pstl offload option,
```

Comment:
Could you please also mention that this test is a second TU for usm_memory_replacement.pass.cpp?

Reply:
Done.